lib/src/kvs/tests/mod.rs

```diff
@@ -18,9 +18,9 @@ pub(crate) enum Kvs {
 
 type ClockType = Arc<RwLock<SizedClock>>;
 
-#[cfg(feature = "kv-tikv,kv-fdb")]
+#[cfg(any(feature = "kv-tikv" feature="kv-fdb"))]
```

Comment:
You are missing a `,`

```rust
#[cfg(any(feature = "kv-tikv", feature="kv-fdb"))]
```

Reply:
Yes, solved now. But the multiread test is failing on transaction conflicts for read 🤯 😂 Thanks though